app/src/main/java/org/mifos/mobile/ui/activities/SettingsActivity.kt

```diff
@@ -6,16 +6,19 @@ import android.os.Bundle
 import androidx.core.app.ActivityCompat
 
 import org.mifos.mobile.R
+import org.mifos.mobile.databinding.ActivitySettingsBinding
 import org.mifos.mobile.ui.activities.base.BaseActivity
 import org.mifos.mobile.ui.fragments.SettingsFragment
 import org.mifos.mobile.utils.Constants
 
 class SettingsActivity : BaseActivity() {
-
+    private lateinit var binding : ActivitySettingsBinding
     private var hasSettingsChanged = false
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_settings)
+        binding = ActivitySettingsBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
         setToolbarTitle(getString(R.string.settings))
         showBackButton()
```

Comment:
with these in place we wont be able to see the toolbar and the backbutton in the darkmode.

Reply:
yes, but I merged your changes into mine for review  and I wasnt able to see the toolbar in dark mode

Reply:
![WhatsApp Image 2023-05-20 at 00 36 17](https://github.com/openMF/mifos-mobile/assets/90026952/619076d1-bca1-4d84-be18-ffc144aba31c)

Reply:
What could be the issue, because the video I shared is for this same PR.